meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging-mfgtool_%.bbappend

```diff
@@ -0,0 +1 @@
+include u-boot-ti-staging.inc
```

Comment:
Can't we have this line as part of the u-boot-ti-staging-mfgtool_2023.04.bb recipe and drop this append?

Still unclear why we need this append since we're maintaining u-boot-ti-staging-mfgtool_2023.04.bb.

Reply:
fixed, thanks!